manipulating-objects/from-list-to-object.js

```diff
@@ -7,12 +7,21 @@
 //   year : 1964
 // }
 
-function fromListToObject(/* your code here */) {
-    //  your code here
+function fromListToObject(arr) {
+  const obj = {};
+  arr.forEach((pair) => {
+    const key = pair[0];
+    const value = pair[1];
+    obj[key] = value;
+  });

```

Comment:
you could destruct in the parameter
```suggestion
  arr.forEach(([key, value]) => {
    obj[key] = value;
  });

```

Reply:
or we could simply use `.fromEntries` method and kill the `obj` variable

```suggestion
return Object.fromEntries(arr);
```

Reply:
Thanks for the tips, looks way better!